saheedt/Ephemeris
# #162335446: Properly set up eslint on project

#### What does this PR do
- properly set up eslint on project

#### Description of Task to be completed?
- add dev dependencies to properly enable linting
- specify react version in eslint config

#### How should this be manually tested
- Remove a semi-colon from a js file in the client folder and linting error should be flagged
- run `npm run lint` and the error should be corrected

#### What are the relevant pivotal tracker stories?
[#162335446](https://www.pivotaltracker.com/story/show/162335446)
